mslib/msui/_tests/test_mss_pyui.py

```diff
@@ -107,7 +107,7 @@ class Test_MSSSideViewWindow(object):
     save_csv = os.path.join(ROOT_DIR, "example.csv")
     save_ftml = os.path.join(ROOT_DIR, "example.ftml")
     save_ftml = save_ftml.replace('\\', '/')
-    save_txt = os.path.join(ROOT_DIR, "example.txt")
+    save_txt = os.path.join(ROOT_DIR, "example.Text")
```

Comment:
why?

Reply:
this is currently to satisfy tests. Import/Export/SaveAs were unified based on the extension which makes trouble when we have multiple times same exension, e.g. save_as_csv. It did in the past the mapping by the extension and not by the key e.g. DLR had than loaded the other csv method. 

I use now the key as identifier to destiguish plugins. The Key for the builtin .txt extension is Text in our setup.

By this changes we can have multiple times plugins which create csv or others with same extension. Also  on reading a file we will first select the Type and then choose the file. So we get  always the right function used

I recognized than that the mscolap Operation Import Feature needs a UI and API change, because there is the mapping to the type missing. So also all tests for this feature needs a refactoring. select first type than export by the prefered extension. 

This was currently a mimum change not to have using the UI for this. Test needs more changes.